### PR TITLE
Form validation

### LINKED ISSUE #16: Bug

### PR DESCRIPTION
**Pull request by johnkioko**
**Form Validation:**
1. Add form validation script on the JS file
2. Add a span on the html with a class of error message to display the error message
3. added styles the span

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,164 +1,197 @@
...
-        <!-- <img src="images/menu_icon.svg" alt="menu icon image"> -->
-        <span class="bar"></span>
-        <span class="bar"></span>
-        <span class="bar"></span>
-      </div>
-   
-    </nav>
-  </header> 
+            <div class="hamberg">
+              <!-- <img src="images/menu_icon.svg" alt="menu icon image"> -->
```
Comment (ahmetbozaci) [inline index.html]: - [ ] To make clean code kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=Follow%20our%20list%20of%20best%20practices%20for%20HTML%20%26%20CSS.)

File: index.html [hunk 2]
```diff
@@ -1,164 +1,197 @@
...
-  <!-- <section class="desktop-mid-section"> -->
-    <p class="header-section-desktop-paragraph">I can help you build a product, 
-      feature or website  <br>
-      Look through some of my work and experience! If<br>
-        you like what you see and have a project you need <br>
-           coded,  don't hestiate to contact me.
-    </p>
+          </p>
+          <!-- paragraph - desktop version -->
+          <!-- <section class="desktop-mid-section"> -->
```
Comment (ahmetbozaci) [inline index.html]: - [ ] To make clean code kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=Follow%20our%20list%20of%20best%20practices%20for%20HTML%20%26%20CSS.)

File: index.html [hunk 3]
```diff
@@ -1,164 +1,197 @@
...
-          <h3>Languages</h3>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
+          <div class="devider"></div>
+          <div class="circle-2">
+            <!-- <img  src="./images/circle-2.svg" alt="background circle 2 desktop" /> -->
```
Comment (ahmetbozaci) [inline index.html]: - [ ] To make clean code kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=Follow%20our%20list%20of%20best%20practices%20for%20HTML%20%26%20CSS.)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @johnkioko  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You added validation to the email with javascript.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️✔️

## Hi @johnkioko 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️
- form works well✔️
- Validations correctly implemented✔️
- No linter errors✔️


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**